src/schemas.test.ts

```diff
@@ -109,6 +109,14 @@ describe('Schema validation', () => {
         expect(validationResult.error).toBe(undefined)
       }
     })
+
+    it('includes the bridge in the token name if present', () => {
```

Comment:
this seems like a slightly fragile way of getting the test coverage, because it would break if we dropped support for that particular token. how about a unit test for `transformCeloTokensForRTDB` instead?